src/components/ToDoItem.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import './ToDoItem.css'
+
+function ToDoItem(props) {
```

Comment:
1. Please use arrow functions when it's possible. Example:
const ToDoItem = () => {
...
}; 

2. Please use ES6 destruction for props. Example:
const ToDoItem = ({text}) => ...

3. Please read about explicit return. Example:
const ToDoItem = ({text}) => (
  <div className="todo-item">
			<input type="checkbox" />
			<h3>{props.text}</h3>
		</div>
);

Reply:
I fixed tabulation and redo funcs into array funcs. Will learn about destruction for props and explicit return